src/lib/TwaManifest.ts

```diff
@@ -66,14 +66,29 @@ function generatePackageId(host) {
  * splashScreenFadeOutDuration: 300
  *
  */
-class TwaManifest {
-  constructor(data) {
+export class TwaManifest {
+  packageId: string;
+  host: string;
+  name: string;
+  themeColor: Color;
+  navigationColor: Color;
+  backgroundColor: Color;
+  enableNotifications: boolean;
+  startUrl: string;
+  iconUrl: string;
+  maskableIconUrl: string;
+  useBrowserOnChromeOS: boolean;
+  splashScreenFadeOutDuration: number;
+  signingKey: string;
+  appVersion: string;
+
+  constructor(data: any) {
```

Comment:
Could you add a TODO comment to change `any` to something else once we've TypeScript-ified the rest of the code?